crates/svm-app/src/types/wasm_value.rs

```diff
@@ -7,3 +11,47 @@ pub enum WasmValue {
     /// A 64-bit integer.
     I64(u64),
 }
+
```

Comment:
I'm ambivalent about whether this is the right place for this piece.
I know it's targeted for the SVM CLI but besides that, I don't see any useful scenario for this.

I'd rather move it to the `svm-cli` crate.